src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -207,13 +209,22 @@ def generate_FAD_1_default_config(self, filename):
         esc_sensors = []
         esc_sensors.append(json.load(
               open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json'))))
+
+        # SAS Harness Test
+        sas_harness_config = {
+			'sasTHName': 'SAS-TH-2',
+			'url': 'https://localhost:9003/v1.2',
+			'serverCert': getCertFilename("server.cert"),
+			'serverKey': getCertFilename("server.key")}
+		
         config = {
             'registrationRequests': devices,
             'conditionalRegistrationData': conditionals,
             'grantRequests': grants,
             'ppas': ppas,
-            'pals': pals,
-            'escSensors' : esc_sensors
+            'palRecords ': pals,
+            'escSensors' : esc_sensors,
+			'sasHarness' : sas_harness_config
```

Comment:
Please fix indentation, here and anywhere else (I'll stop commenting on it after this point).

As a note, it's generally better to use spaces instead of tabs since they will be rendered consistently on all platforms. It's probably better not to do it in this PR, though, so that we can get it out the door sooner and to ease reviewing. If we want to update, we could do it in a followup PR that would be very fast to review.

Reply:
Re-commenting in case this was missed.